src/aggregate.d

```diff
@@ -126,7 +126,7 @@ public:
             sc2.inunion = 1;
         sc2.protection = Prot(PROTpublic);
         sc2.explicitProtection = 0;
-        sc2.structalign = STRUCTALIGN_DEFAULT;
+        sc2.aligndecl = null;
```

Comment:
My strong reason is, `structalign` was a name for the alignment *value*, and now `aligndecl` is for the `AlignDeclaration` where is inner-most scope. They're completely different things. Using appropriate name is necessary for self-documented code.